pkg/errors/release.go

```diff
@@ -194,3 +194,24 @@ func NewNotWorkingOnStrategyError(contenderReleaseKey string) NotWorkingOnStrate
 		contenderReleaseKey: contenderReleaseKey,
 	}
 }
+
+type ContenderStrategyIncompleteError struct {
+	relKey string
+}
+
+func (e ContenderStrategyIncompleteError) Error() string {
+	return fmt.Sprintf(
+		"release contender %q strategy is incomplete yet",
+		e.relKey,
+	)
+}
+
+func (e ContenderStrategyIncompleteError) ShouldRetry() bool {
+	return true
+}
```

Comment:
I'm not sure if retries here are appropriate. I'd expect that when the contender achieves its desired target step, it would re-enqueue its neighbors, making the blind retries redundant.